recipes/scikit-optimize/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "scikit-optimize" %}
+{% set version = "0.3" %}
+{% set bundle = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash = "d8934c33e5958227d2ff97a4fae2134d2a00432b99e7f9d90b17e45cb1da9bb5" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ bundle }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ bundle }}
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: {{ build }}
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - numpy
+    - scipy
+    - scikit-learn >=0.18
+    - matplotlib
+
+test:
+  imports:
+    - skopt
+    - skopt.learning
+    - skopt.learning.gaussian_process
+    - skopt.optimizer
+    - skopt.space
+
+about:
+  home: https://scikit-optimize.github.io/
+  license_file: {{ RECIPE_DIR }}/LICENSE.md
```

Comment:
My impression was that @jakirkham was asking for the url to be embedded in the file for recipes he was reviewing and I was following that lead. @jakirkham, correct me if I'm wrong. Generally though, I think it's a good idea since feedstocks get updated infrequently and having the reminder there to check to see if the missing file has since been included helps. I tend to be picky since it's often the last time anyone but the maintainer will ever look at the recipe.